### DISCUSSION TITLE
How to move SimpleLogin existing account

### DISCUSSION THREAD

**Discussion by markusqw1** [simple-login/app#1790] [Self-hosting question] [answered]
Hi I'm considering self hosting and I am wondering if I can use my existing SimpleLogin premium account when going to self hosting or what happens to my existing account.

**Comment by nguyenkims** [comment, marked as answer]
@markusqw1 you can't move aliases that use SimpleLogin domains as these domains don't point to your server. You can however move all aliases that use your own domain.
